Logging/DeprecationHandler.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of Ekino New Relic bundle.
+ *
+ * (c) Ekino - Thomas Rabaix <thomas.rabaix@ekino.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Ekino\Bundle\NewRelicBundle\Logging;
```

Comment:
I'm not sure about the namespace. Any suggestion?

Reply:
Maybe we should consider this as a "Listener" by moving it to the "Listener" namespace?